lua/rzls/handlers/providehtmldocumentcolor.lua

```diff
@@ -0,0 +1,39 @@
+local documentstore = require("rzls.documentstore")
+local razor = require("rzls.razor")
+local Log = require("rzls.log")
+
+local empty_response = {}
+
+---@param _err lsp.ResponseError
+---@param result lsp.DocumentDiagnosticParams
```

Comment:
You can define a type above like this
```lua
---@class rzls.provideHtmlDocumentColorParams
---@field textDocument lsp.TextDocumentIdentifier
---@field _razor_hostDocumentVersion integer
```

Then use that instead of `lsp.DocumentDiagnosticParams`